x-pack/plugins/security_solution/common/api/endpoint/actions/list_route.ts

```diff
@@ -34,6 +35,30 @@ export const EndpointActionListRequestSchema = {
         schema.string({ minLength: 1 }),
       ])
     ),
+    agentTypes: schema.maybe(
```

Comment:
Suggestion: rename it to `agentType` to be consistent with other usages of this value already in place

Reply:
Yeah I started with agentType and then changed it to plural as we use plurals for all other params that accept multiple values.

Reply:
ahhh. ok. did not realize that. agreed that you should leave it as is. Thanks